lib/scala/interpreter-dsl/src/main/java/org/enso/interpreter/dsl/model/MethodDefinition.java

```diff
@@ -198,7 +198,7 @@ public static class ArgumentDefinition {
     private static final String THUNK = "org.enso.interpreter.runtime.callable.argument.Thunk";
     private static final String CALLER_INFO = "org.enso.interpreter.runtime.callable.CallerInfo";
     private static final String DATAFLOW_ERROR = "org.enso.interpreter.runtime.error.DataflowError";
-    private static final String THIS = "this";
+    private static final String THIS = "self";
```

Comment:
Don't we want to rename `THIS` to `SELF` too?

Reply:
It doesn't hurt yes, although this is Java-specific

Reply:
Ah, in that case maybe indeed it's better to keep it as-is.